goss/goss.yaml

```diff
@@ -24,6 +24,11 @@ command:
     stdout:
       - 21-beta
     stderr: []
+  playwright --version:
+    exit-status: 0
+    stdout:
+      - 1.37.1
```

Comment:
seem not to be the value received : 
```
Command: playwright --version: exit-status:
[2023-09-13T16:25:14.233Z] 2023-09-13T16:25:14Z:     azure-arm.ubuntu: Expected
[2023-09-13T16:25:14.233Z] 2023-09-13T16:25:14Z:     azure-arm.ubuntu:     <int>: 127
[2023-09-13T16:25:14.233Z] 2023-09-13T16:25:14Z:     azure-arm.ubuntu: to equal
[2023-09-13T16:25:14.233Z] 2023-09-13T16:25:14Z:     azure-arm.ubuntu:     <int>: 0
[2023-09-13T16:25:14.233Z] 2023-09-13T16:25:14Z:     azure-arm.ubuntu: Command: playwright --version: stdout: patterns not found: [1.37.1]
```

Reply:
I've approved your suggested change. Does it work now?

Reply:
it mean changing also the updatecli manifest and then the dot at `.asdf` make it difficult for key name ... I suggest to change the goss.yaml to use the new version with exec as here : https://github.com/goss-org/goss/blob/master/docs/manual.md#command

but then we will have to update the updatecli manifest for jdk that are updating that file

Reply:
you can try locally with this : https://github.com/jenkins-infra/packer-images/blob/main/README.md#invoking-the-build-locally